pkg/security/scc/byrestrictions.go

```diff
@@ -25,19 +26,29 @@ func (s ByRestrictions) Less(i, j int) bool {
 type points int
 
 const (
-	privilegedPoints points = 20
+	privilegedPoints points = 200000
 
-	hostVolumePoints       points = 10
-	nonTrivialVolumePoints points = 5
+	hostVolumePoints       points = 100000
+	nonTrivialVolumePoints points = 50000
 
-	runAsAnyUserPoints points = 4
-	runAsNonRootPoints points = 3
-	runAsRangePoints   points = 2
-	runAsUserPoints    points = 1
+	runAsAnyUserPoints points = 40000
+	runAsNonRootPoints points = 30000
+	runAsRangePoints   points = 20000
+	runAsUserPoints    points = 10000
+
+	runCapsDefaultPoints  points = 5000
+	runCapsAddOnePoints   points = 300
+	runCapsAllowAllPoints points = 4000
+	runCapsAllowOnePoints points = 10
```

Comment:
> I would expect defaults to weigh little if they are to weigh anything at all.

If they will have zero weight then SCC that grants 3 caps by default will have the same weight as the SCC that grants 1 cap by default.

I'd expect that default, add and removed caps have the same weight because they're operate on the same entity (capability). In the end we're trying to sort by a number of the caps that pod will have after being admitted by an SCC.